cypress/e2e/buttonLoad.cy.tsx

```diff
@@ -0,0 +1,24 @@
+describe('Buttons Reload', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.contains('button', 'Conectar con una clave secreta').click();
+    cy.get('.check__input').as('checkboxInput');
+    cy.contains('button', 'Continuar').as('buttonContinue');
+    cy.get('@checkboxInput').check();
+    cy.get('@buttonContinue').click();
+    cy.get('input[type="password"]').type(Cypress.env('secret_key'));
+    cy.get('button[class="button button__complete"]').click();
+    cy.wait(3000);
+  });
+
+  it('Should see a Your balance has been updated message when you click the reload payments button', () => {
```

Comment:
Should show a message saying Your Balance has been updated when reload button is clicked